reminder/util.py

```diff
@@ -43,7 +44,10 @@ def __init__(self, name: str, label: str = None, *, required: bool = False):
 
     def match(self, val: str, evt: MessageEvent = None, instance: 'ReminderBot' = None
               ) -> Tuple[str, Optional[datetime]]:
-        tz = pytz.UTC
+        try:
+            tz = pytz.timezone(instance.config["timezone"])
+        except (pytz.UnknownTimeZoneError, StopIteration, IndexError):
+            tz = pytz.UTC
         use_locales = [locales["en_iso"]]
         if instance:
             tz = instance.db.get_timezone(evt.sender)
```

Comment:
This method will override any default set earlier. The database class probably doesn't have access to the `ReminderBot` instance, but you could update `get_timezone` to have a parameter for the default value.